addons/base_import_module/models/ir_module.py

```diff
@@ -115,7 +121,7 @@ def _import_module(self, module, path, force=False):
                 directive, target, path = IrAsset._process_command(command)
                 path = path if path.startswith('/') else '/' + path # Ensures a '/' at the start
                 assets_vals.append({
-                    'name': f'{module}.{bundle}.{path}',
+                    'name': f'{module}_{bundle}_{path}'.replace('.', '_'),
```

Comment:
as you change the name, could it cause issues with the `search` below ?

Reply:
oups I made the change at the wrong place